fbw-a32nx/src/systems/fmgc/src/components/EfisLabels.ts

```diff
@@ -17,14 +17,25 @@ export class EfisLabels implements FmgcComponent {
         const transitionAltitude = this.flightPlanManager.originTransitionAltitude;
         const transitionLevel = this.flightPlanManager.destinationTransitionLevel;
 
-        // FIXME ARINC429 when the PR adding a TS impl. lands...
         if (transitionAltitude !== this.lastTransitionAltitude) {
-            SimVar.SetSimVarValue('L:AIRLINER_TRANS_ALT', 'Number', transitionAltitude ?? 0);
+            Arinc429Word.toSimVarValue(
+                'L:A32NX_FM1_TRANS_ALT', transitionAltitude ?? 0, transitionAltitude !== undefined ? Arinc429SignStatusMatrix.NormalOperation : Arinc429SignStatusMatrix.NoComputedData,
+            );
+            Arinc429Word.toSimVarValue(
+                'L:A32NX_FM2_TRANS_ALT', transitionAltitude ?? 0, transitionAltitude !== undefined ? Arinc429SignStatusMatrix.NormalOperation : Arinc429SignStatusMatrix.NoComputedData,
+            );
+
             this.lastTransitionAltitude = transitionAltitude;
         }
 
         if (transitionLevel !== this.lastTransitionLevel) {
-            SimVar.SetSimVarValue('L:AIRLINER_APPR_TRANS_ALT', 'Number', (transitionLevel ?? 0) * 100);
+            Arinc429Word.toSimVarValue(
+                'L:A32NX_FM1_TRANS_LVL', transitionLevel ?? 0, transitionLevel !== undefined ? Arinc429SignStatusMatrix.NormalOperation : Arinc429SignStatusMatrix.NoComputedData,
+            );
+            Arinc429Word.toSimVarValue(
+                'L:A32NX_FM2_TRANS_LVL', transitionLevel ?? 0, transitionLevel !== undefined ? Arinc429SignStatusMatrix.NormalOperation : Arinc429SignStatusMatrix.NoComputedData,
+            );
+
```

Comment:
These should all go FW when the respective FM is not powered. The old FMC has some stuff that takes care of handling the bus outputs automatically, including proper quantisation and range limiting per the word encoding, but nothing exists yet in the TS code.